In [4]:
import pandas as pd

In [5]:
df = pd.read_csv("../data/yerba-mate-1970-2024.csv", encoding="latin-1")

In [6]:
produccion_anual = df.groupby("anio")["produccion_tm"].sum()

produccion_1970 = produccion_anual.loc[1970]
produccion_2024 = produccion_anual.loc[2024]

crecimiento = ((produccion_2024 - produccion_1970) / produccion_1970) * 100

crecimiento

round(crecimiento, 2)

np.float64(1185.59)

# Insight 1 - Historical Production Growth

Total yerba mate production grew by 1,185.59% between 1970 and 2024.

This extraordinary growth indicates a strong expansion of the yerba mate sector in Argentina over the past five decades.

This increase may be associated with multiple factors, including:

-expansion of cultivated area

-improvements in agricultural techniques

-higher domestic and international demand

-professionalization of the sector

However, the time series analysis reveals a significant breakpoint in 1998, where an abrupt increase in recorded values is observed. This jump is not necessarily due to an actual and immediate rise in physical production; this will be discussed further in 'Insight 4'.

This result confirms that yerba mate is an agricultural sector with sustained growth and considerable economic importance.

In [7]:
produccion_provincia = df.groupby("provincia")["produccion_tm"].sum()

total_produccion = produccion_provincia.sum()

participacion = (produccion_provincia / total_produccion) * 100

participacion

provincia
Corrientes    12.969549
Misiones      87.030451
Name: produccion_tm, dtype: float64

# Insight 2 - High Geographic Concentration of Production

The analysis reveals a strong geographic concentration of yerba mate production in Argentina.

Misiones accounts for 87.03% of total national production, while Corrientes represents only 12.97%.

This indicates that the Argentine yerba mate industry depends almost exclusively on the province of Misiones.

This concentration implies:

-high regional specialization

-production dependence on a single province

-potential risk from regional climatic or economic events

At the same time, it confirms the strategic role of Misiones as the country’s main yerba mate production hub.

In [8]:
rendimiento_anual = df.groupby("anio")["rendimiento_kgxha"].mean()

rendimiento_1970 = rendimiento_anual.loc[1970]
rendimiento_2024 = rendimiento_anual.loc[2024]

crecimiento_rendimiento = ((rendimiento_2024 - rendimiento_1970) / rendimiento_1970) * 100

crecimiento_rendimiento

np.float64(138.99884502309953)

# Insight 3 - Significant Improvement in Production Efficiency

The average yield of yerba mate increased by 139.00% between 1970 and 2024.

This means that today more than twice as much is produced per hectare compared to five decades ago.

This increase in production efficiency suggests important improvements in:

-agricultural techniques

-genetic quality of the plants

-crop management

-use of agricultural technology

This yield growth indicates that the increase in total production is not solely due to the expansion of cultivated area, but also to a significant improvement in productivity. However, it is important to consider the points discussed in 'Insight 4'.

In [10]:
produccion_anual = df.groupby("anio")["produccion_tm"].sum()

antes = produccion_anual.loc[1970:1997].mean()
despues = produccion_anual.loc[1998:2024].mean()

cambio_promedio = ((despues - antes) / antes) * 100

round(cambio_promedio, 2)

np.float64(374.66)

# Insight 4 - Breakpoint in 1998 Due to Mandatory Formalization of Production

The 374.66% increase in average annual production from 1998 does not necessarily reflect a real growth in yerba mate production, but rather a change in the registration and documentation system.

In the late 1990s and early 2000s, the Argentine government implemented measures that made the reporting of produced tons stricter and mandatory. This process was consolidated with the creation of the National Yerba Mate Institute, which established formal mechanisms for control, registration, and monitoring of production, especially in the province of Misiones.

Therefore, the jump observed in the data primarily reflects an improvement in the formalization and traceability of information, rather than a sudden increase in actual production.